optuna/cli.py

```diff
@@ -215,7 +215,10 @@ def _dump_table(records: list[dict[str, Any]], header: list[str]) -> str:
         for t in value_types:
             if t == ValueType.STRING:
                 value_type = ValueType.STRING
-        max_width = max(len(header[column]), max(row[column].width() for row in rows))
+        if not rows:
```

Comment:
```suggestion
        if len(rows) == 0:
```
Please use `len(rows) == 0` for checking if the list is empty, rather than relying on `not rows`.